<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page3</class>
 <widget class="QWidget" name="Page3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IP List</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2d3242;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">margin: 0;
padding: 0;
color: white;</string>
     </property>
     <widget class="QPushButton" name="blockedButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;components/page2/images/blocked_on.png&quot;);
margin: 0;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="allowedButton">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>0</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(&quot;components/page2/images/allowed_off.png&quot;);
margin: 0;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>0</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="delButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>0</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>삭제</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="blockedTableView">
     <property name="styleSheet">
      <string notr="true">QTableView {
    gridline-color: rgb(50, 55, 100);
    alternate-background-color: rgb(68, 64, 122);
	padding: 0;
	margin: 0;
	border: 0;
}
QTableView::item {
    padding: 5px;
    border-bottom: 1px solid rgb(68, 64, 122);
}
QHeaderView::section {
    background-color: rgb(68, 64, 122);
    color: white;
    padding: 5px;
    border: none;
    font-weight: bold;
}
QTableView::item:selected {
    background-color: rgb(70, 77, 140);
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
